### PR TITLE
Update 05_multistep_td.py

### PR DESCRIPTION
**Pull request by Cohya**
Fixing minor issue

### CODE REVIEW

File: 05_multistep_td.py [hunk 1]
```diff
@@ -172,14 +172,15 @@ def train(self):
             self.n_step_buffer.append(temp_transition)
             if len(self.n_step_buffer) == self.n_step:  # fill the n-step buffer for the first translation
                 # add a multi step transition
-                reward, next_obs, done = self.get_n_step_info(self.n_step_buffer, self.gamma)
+                reward, next_obs_to_store, done = self.get_n_step_info(self.n_step_buffer, self.gamma)
                 obs, action = self.n_step_buffer[0][:2]
-
+            else:
```
Comment (Huixxi) [inline 05_multistep_td.py:177]: can you make some explains about that improvement?

File: 05_multistep_td.py [hunk 2]
```diff
@@ -204,7 +205,7 @@ def train_step(self):
         target_q = self.get_target_value(self.b_next_states)    # get target q-value through the target network
         # get td_targets of batch states
         td_target = self.b_rewards + \
-            self.gamma * target_q[np.arange(target_q.shape[0]), best_action_idxes] * (1 - self.b_dones)
+            self.gamma**(self.n_step ) * target_q[np.arange(target_q.shape[0]), best_action_idxes] * (1 - self.b_dones)
```
Comment (Huixxi) [inline 05_multistep_td.py:208]: and this place (btw, it has a unused blank)

**Comment by Huixxi** [comment]
> Fixing minor issue

Add some comments to explain your improvement is best~

File: 05_multistep_td.py (hunk 1 above)
Comment (Cohya) [reply 05_multistep_td.py:177]: It takes care of the cases where "done =True" is in the queue.
In that case the "next observation" is the "done state" and not the "real next observation".

The "obs, action = self.n_step_buffer[0][:2] : it is for the storage: You need to store the action and obs in time = t-n

File: 05_multistep_td.py (hunk 2 above)
Comment (Cohya) [reply 05_multistep_td.py:208]: TD =0 : G = r+ gamma** argmax_{a'}Q(s',a')

n_steps: G = r  + gamma* r(t+1) + gammer^2 r(t+2) + gamma^3 argmax_{a}Q(s_{t+n},a)

In your code self.b_rewards = = r  + gamma* r(t+1) + gammer^2 r(t+2)

Comment (Huixxi) [reply 05_multistep_td.py:208]: Ok, I got it, just need to remove the redundant blank in "self.n_step "

**Pull request closed without merging**